source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +478,35 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of entity names to an an array of
+    event names.
+    
+    For each entity name, the test runner MUST create the respective entity
+    with a type of "event list". If the entity already exists
```

Comment:
Since [Supported Entity Types](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#supported-entity-types) exhaustively lists what we're allowed to store in the entity map, I suggest you refer to "BSON array" here. The following paragraph should then make it clear that we'll be appending documents to this array.

I'm not sure if some folks might have read "event list" here and assumed they could use a thread-safe list object. I suppose there's nothing wrong with doing so as long as it's ultimately converted to a BSON array before we expose it to the workload executor.

Alternatively, if you want to allow flexibility here (e.g. let drivers use a thread-safe list and never convert it to a BSON array), I'd suggest adding "event list" as an explicitly supported type. There, you'd mention that drivers can implement it as they want and I'd suggest providing a thread-safe list or BSON array as possibilities.

Reply:
I added the event list entity as you suggested.